src/common/components/clusterConfiguration/utils.ts

```diff
@@ -59,7 +59,7 @@ export const isAdvNetworkConf = (cluster: Cluster, defaultNetworkSettings: Clust
   cluster.clusterNetworkCidr !== defaultNetworkSettings.clusterNetworkCidr ||
   cluster.clusterNetworkHostPrefix !== defaultNetworkSettings.clusterNetworkHostPrefix ||
   cluster.serviceNetworkCidr !== defaultNetworkSettings.serviceNetworkCidr ||
-  cluster.networkType !== 'OpenShiftSDN';
+  (Boolean(cluster.networkType) && cluster.networkType !== 'OpenShiftSDN');
```

Comment:
nit: `(!!cluster.networkType && cluster.networkType !== 'OpenShiftSDN')`

Reply:
I think we both agree that `!!x` and `Boolean(x)` are the same. BTW, nowadays in terms of performance, they are also the same.
Perhaps we can agree that we disagree on this, but don't you think using `Boolean(x)` is more explicit? (specially helpful for newcomers)

Reply:
I think we both agree that it's better to agree on those little things which can easily generate a disagreement.